cmd/loop/main.go

```diff
@@ -256,16 +280,79 @@ func logSwap(swap *looprpc.SwapStatus) {
 	fmt.Println()
 }
 
-func getClientConn(address string) (*grpc.ClientConn, error) {
+func getClientConn(address, tlsCertPath, macaroonPath string) (*grpc.ClientConn,
+	error) {
+
 	opts := []grpc.DialOption{
-		grpc.WithInsecure(),
 		grpc.WithDefaultCallOptions(maxMsgRecvSize),
 	}
 
+	switch {
+	// If a TLS certificate file is specified, we need to load it and build
+	// transport credentials with it.
+	case tlsCertPath != "":
```

Comment:
Maybe it is a good idea to first do a preparatory PR that adds TLS and macaroons to `loop`? That would also fix #194 and #13

Reply:
We can do this later, this isn't in the scope of this PR.

Reply:
Discussed offline